README.md

```diff
@@ -1,15 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# JavaScript Capstone 2
 
-> Description the project.
+> A javascript application that displays movies and TV shows from the TVMAZE API
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Javascript
+- Webpack
+- HTML&CSS
 
 ## Live Demo (if available)
 
```

Comment:
- [ ] The Live demo link is broken. If the link is not available, I recommend removing that to make your README more professional.